dm/tests/many_tables/run.sh

```diff
@@ -16,6 +16,7 @@ function prepare_data() {
 		for j in $(seq 2); do
 			run_sql "INSERT INTO many_tables_db.t$i VALUES ($j,${j}000$j),($j,${j}001$j);" $MYSQL_PORT1 $MYSQL_PASSWORD1
 		done
+		run_sql "INSERT INTO many_tables_db.t$i VALUES (9, 90009);" $MYSQL_PORT1 $MYSQL_PASSWORD1
```

Comment:
changes of this file should not be included

Reply:
Have removed this part of the changes.